hoover3_client/src/pages/admin/dataaccess.rs

```diff
@@ -0,0 +1,119 @@
+use dioxus::prelude::*;
+use hoover3_types::data_access::DataAccessSettings;
+use std::path::PathBuf;
+
+use crate::api::create_or_update_data_access_settings;
+
+#[component]
+pub fn ConfigureDataAccessForm(on_close: Callback) -> Element {
+    let mut root_path = use_signal(|| "".to_string());
+    let mut s3_bucket = use_signal(|| "".to_string());
+    let mut s3_access_key = use_signal(|| "".to_string());
```

Comment:
move these to a separate struct in a separrate table with a separate unique id, to allow multiple s3/webdav servers